docinfo-footer.html

```diff
@@ -1,3 +1,49 @@
 <div id="footer-github">
     Edit these docs <a href="https://github.com/LibrePCB/librepcb-doc/">on GitHub</a>!
 </div>
+<script type="text/javascript">
+  function tocLinkCloseAll() {
+    Array.from(document.querySelectorAll(".toc-visible")).forEach(function(item) {
+      item.classList.remove("toc-visible");
+      item.classList.add("toc-invisible");
+    });
```

Comment:
I think you can also do this with a for-of loop:

```javascript
for (const item of document.querySelectorAll(".toc-visible")) {
      item.classList.remove("toc-visible");
      item.classList.add("toc-invisible");
}
```

Also, I'd extract the two class names into constants.

Reply:
To be honest, it is my first ES2015 without using Babel. Thanks for the tip.

Reply:
Well, kudos to you. I casually talked to some colleagues about the existence of `for-of` and they have been discussing about it for the past 30 minutes or so... 😃